_internals/configuration.md

```diff
@@ -481,8 +481,12 @@ Set `TEXTSIZE`.
 
 ### `babelfishpg_tsql.use_antlr`
 
-Selects the new ANTLR parser for pl/tsql functions, procedures, trigger, and
-batches. The default value is `true`.
+This variable was added for historically reasons, and it will be removed in future releases. 
```

Comment:
How about:

This parameter is now obsolete, and will be removed in a future release.

Reply:
Thanks for the review, changes applied.